### PR TITLE
Refactor to SendBitcoin

### PR DESCRIPTION
**Pull request by ronaldsg20**
I've created the sendBitcoin component as a container of
                bitcoinForm interactions. Also I grouped all the Trezor
                and Ledger behavior on one container.

### CODE REVIEW

File: src/components/create/SendBitcoin.vue [hunk 1]
```diff
@@ -87,61 +82,44 @@ export default class SendBitcoinLedger extends Vue {
...
-  feeBTC: SatoshiBig = new SatoshiBig('0', 'satoshi');
-
-  txBuilder: LedgerTxBuilder = new LedgerTxBuilder();
+  txBuilder!: TxBuilder;
 
   @State('pegInTx') peginTxState!: PegInTxState;
 
   @Action(constants.PEGIN_TX_START_ASKING_FOR_BALANCE, { namespace: 'pegInTx' }) startAskingForBalanceStore !: () => Promise<void>;
 
+  @Action(constants.IS_TREZOR_CONNECTED, { namespace: 'pegInTx' }) setTrezorConnected !: any;
```
Comment (josedahlquist) [inline src/components/create/SendBitcoin.vue:91]: what's this for? We shouldn't have wallet specific behavior here

File: src/components/create/SendBitcoin.vue [hunk 2]
```diff
@@ -218,15 +202,23 @@ export default class SendBitcoinLedger extends Vue {
     return currentComponent;
   }
 
+  @Emit()
+  setTxBuilder():void {
```
Comment (josedahlquist) [inline src/components/create/SendBitcoin.vue:206]: Same question here... why do we need wallet specific txBuilders ? I insist on reconsidering changing the definition of txBuilder as this seems a bad practice

File: src/components/create/SendBitcoin.vue [hunk 3]
```diff
@@ -235,21 +227,17 @@ export default class SendBitcoinLedger extends Vue {
...
-      inputs: [],
-      outputs: [],
-    };
-    this.amount = new SatoshiBig('0', 'satoshi');
-    this.refundAddress = '';
-    this.recipient = '';
-    this.feeBTC = new SatoshiBig('0', 'satoshi');
-    this.txBuilder = new LedgerTxBuilder();
+    this.setTxBuilder();
+    await this.walletService.stopAskingForBalance();
```
Comment (josedahlquist) [inline src/components/create/SendBitcoin.vue:231]: Should we have this as a store action instead?

**Comment by Liuminjin** [comment]
You have to synchronize your wallet and then you debug it

File: src/components/create/SendBitcoin.vue (hunk 2 above)
Comment (ronaldsg20) [reply src/components/create/SendBitcoin.vue:206]: I'm going to change the txBuilder to be accesed from the store

File: src/components/create/SendBitcoin.vue (hunk 3 above)
Comment (ronaldsg20) [reply src/components/create/SendBitcoin.vue:231]: yes and the TxBuilder should be a getter on the next PR

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Review by josedahlquist — APPROVED** [review]

**Pull request merged**